app/assets/stylesheets/govuk_publishing_components/components/_image-card.scss

```diff
@@ -3,6 +3,22 @@
   @include govuk-text-colour;
   position: relative;
   margin-bottom: govuk-spacing(6);
+  display: flex;
```

Comment:
IE10 and below doesn't seem to support the `flex` styles here properly, resulting in the title (and text) appearing before the image. Might be able to fix with some custom IE stuff.

<img width="972" alt="Screenshot 2020-09-21 at 15 27 44" src="https://user-images.githubusercontent.com/861310/93779775-3b760f00-fc1f-11ea-93e4-020d1fa061d9.png">

Reply:
I've added -ms prefix versions and it now works in IE10 but not earlier. I think that's fine, the pages are perfectly legible and understandable in the old versions.